examples/trusted_information_retrieval/client/trusted_information_retrieval.cc

```diff
@@ -27,33 +27,28 @@
 
 ABSL_FLAG(std::string, address, "localhost:8080", "Address of the Oak application to connect to");
 ABSL_FLAG(std::string, ca_cert, "", "Path to the PEM-encoded CA root certificate");
-ABSL_FLAG(float, latitude, float{}, "Requested location's latitude in degrees (WGS84)");
-ABSL_FLAG(float, longitude, float{}, "Requested location's longitude in degrees (WGS84)");
+ABSL_FLAG(std::string, id, "", "Database ID of the point of interest");
 
-using ::oak::examples::trusted_information_retrieval::ListPointsOfInterestRequest;
-using ::oak::examples::trusted_information_retrieval::ListPointsOfInterestResponse;
-using ::oak::examples::trusted_information_retrieval::Location;
+using ::oak::examples::trusted_information_retrieval::GetPointOfInterestRequest;
+using ::oak::examples::trusted_information_retrieval::GetPointOfInterestResponse;
 using ::oak::examples::trusted_information_retrieval::PointOfInterest;
 using ::oak::examples::trusted_information_retrieval::TrustedInformationRetrieval;
 
-void get_nearest_point_of_interest(TrustedInformationRetrieval::Stub* stub, float latitude,
-                                   float longitude) {
+void get_point_of_interest(TrustedInformationRetrieval::Stub* stub, std::string id) {
   grpc::ClientContext context;
-  LOG(INFO) << "Getting nearest point of interest:";
-  ListPointsOfInterestRequest request;
-  Location* location = request.mutable_location();
-  location->set_latitude(latitude);
-  location->set_longitude(longitude);
-  ListPointsOfInterestResponse response;
-  grpc::Status status = stub->ListPointsOfInterest(&context, request, &response);
+  LOG(INFO) << "Getting point of interest:";
+  GetPointOfInterestRequest request;
+  request.set_id(id);
+  GetPointOfInterestResponse response;
+  grpc::Status status = stub->GetPointOfInterest(&context, request, &response);
   if (!status.ok()) {
-    LOG(ERROR) << "Could not get nearest point of interest: " << status.error_code() << ": "
+    LOG(ERROR) << "Could not get point of interest: " << status.error_code() << ": "
```

Comment:
Should this be FATAL? (I think I made this same change several times already :P )